test/demo/embed.html

```diff
@@ -0,0 +1,33 @@
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>CartoWall Climbing Topo</title>
```

Comment:
Excellent. You can make a pull request for it here: 

https://github.com/openseadragon/site-build/blob/master/www/in-the-wild.html

...or if you're not up for that, at least add the URL here:

https://github.com/openseadragon/site-build/issues/159

...and hopefully someone else will get around to it.